Example/Podfile

```diff
@@ -1,8 +1,10 @@
+platform :ios, '9.0'
 source 'https://github.com/CocoaPods/Specs.git'
 use_frameworks!
 
 target 'CMHealth_Example' do
   pod 'CMHealth', :path => '../'
+  #pod "CareKit", "~> 1.0"
```

Comment:
This is removed from the podfile, but still present in the Podfile.lock. Is this on purpose?